Cargo.toml

```diff
@@ -1,3 +1,6 @@
 [workspace]
-members = ["compio"]
+members = [
+  "compio",
```

Comment:
I know it is a little annoying but, would you also turn its tab width to 4?

Reply:
Sure